src/app/lib/config/services/japanese.js

```diff
@@ -341,7 +341,7 @@ export const service = {
       },
       {
         title: 'ビデオ',
-        url: '/japanese/video-55128146',
+        url: 'japanese/topics/c132079wln0t',
```

Comment:
This might need a leading forward slash. It might be okay, but the rest of the URL's seem to have them so best to be consistent just in case

Reply:
Doh! sorted. Thanks @HarveyPeachey